### PR TITLE
Modfqmmgr 558

### PR DESCRIPTION
**Pull request by kjain110**

### CODE REVIEW

File: src/main/resources/entity-types/organizations/simple_organizations.json5 [hunk 1]
```diff
@@ -1306,27 +1306,50 @@
...
       visibleByDefault: false,
-      valueGetter: ":sourceAlias.jsonb->>'accounts'",
+      valueGetter: "(SELECT jsonb_agg(record.value || \
+                                    jsonb_build_object('acqUnit', \
+                                    (SELECT jsonb_agg(acq_unit.jsonb ->> 'name') \
+                                     FROM jsonb_array_elements_text(record.value -> 'acqUnitIds') id \
+                                     JOIN src_acquisitions_unit acq_unit \
+                                       ON id.value::text = acq_unit.id::text)) \
+                                   ORDER BY record.value ->> 'acqUnitIds') \
+                 FROM jsonb_array_elements((:sourceAlias.jsonb -> 'accounts')) record(value))::text"
```
Comment (bvsharp) [inline src/main/resources/entity-types/organizations/simple_organizations.json5:1352]: I'm kind of confused by this, shouldn't this be the valueGetter for the entire accounts object?

Comment (kjain110) [reply src/main/resources/entity-types/organizations/simple_organizations.json5:1352]: this is for the entire accounts objects. It's at the column level not at the nested column level. @bvsharp

Comment (mweaver-ebsco) [reply src/main/resources/entity-types/organizations/simple_organizations.json5:1352]: I think this is doing the fun shenanigans where it adds an extra property to the object (which explains why the `property` would be "acqUnit")

Comment (kjain110) [reply src/main/resources/entity-types/organizations/simple_organizations.json5:1352]: This is how we have for contributors and classification as well

**Review by mweaver-ebsco — APPROVED** [review]

**Review by bvsharp — APPROVED** [review]

**Pull request merged**